b0: Add note and warning about flashing on nrf9160 (UICR must be erased) (#1818)
NordicPlayground/fw-nrfconnect-nrf: samples/bootloader/README.rst

Alternatively, to facilitate the manufacturing process of a device with the bootloader sample, it is possible to decouple this process and program the sample HEX file and the HEX file containing the public key hashes separately.
If you choose to do so, use the Python scripts in ``scripts\bootloader`` to create and provision the keys manually.

>    .. note::

Reviewer: This line shouldn't be indented (and the next lines accordingly - three spaces).

Author: This indentation scheme is used elsewhere in this document, e.g. https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1818/files/3dd334ab8b6c24fdffa16477fd864eae1f92d1af#diff-0f7f2103603f132c693d35086cfa585bR106

Should I fix them?

Reviewer: No - there it is part of a step.

      On nRF9160, the provisioning data is held in the OTP region in UICR.

>       Because of this, the UICR must be cleared when flashing the bootlaoder.

Reviewer: ```suggestion
      Because of this, you must clear the UICR before flashing the bootloader.
```

Reviewer: Is "before" correct?

>       This can be done by calling ``west flash`` with the ``--erase`` option.

Reviewer: How do you do it in SES?

Reviewer 2: :upside_down_face:

Author: @ru-fu: I added a sentence about SES, and added a bit more text as well.

Alternatively, to facilitate the manufacturing process of a device with the bootloader sample, it is possible to decouple this process and program the sample HEX file and the HEX file containing the public key hashes separately.
If you choose to do so, use the Python scripts in ``scripts\bootloader`` to create and provision the keys manually.

   .. note::
      On nRF9160, the provisioning data is held in the OTP region in UICR.

>       Because of this, you must erase the UICR before flashing the bootloader.

Reviewer: ```suggestion
      Because of this, you must erase the UICR before programming the bootloader.
```
Sorry, missed this one in the last round. ;)

      On nRF9160, the UICR can only be erased by erasing the whole chip.

>       This can be done by calling ``west flash`` with the ``--erase`` option to erase the whole chip and then flash.

Reviewer: ```suggestion
      To do so on the command line, call ``west flash`` with the ``--erase`` option.
      This will erase the whole chip before programming the new image.
```

>       On SES, choose Target->"Connect J-Link", then Target->"Erase All" to erase the whole chip.

Reviewer: ```suggestion
      In |SES|, choose :guilabel:`Target` > :guilabel:`Connect J-Link` and then :guilabel:`Target` > :guilabel:`Erase All` to erase the whole chip.
```
